scripts/generate-openapi-script.sh

```diff
@@ -14,9 +14,12 @@ BASE_PATH="https://raw.githubusercontent.com/dockstore/dockstore/$npm_package_co
 wget --no-verbose https://repo.maven.apache.org/maven2/org/openapitools/openapi-generator-cli/${GENERATOR_VERSION}/openapi-generator-cli-${GENERATOR_VERSION}.jar -O openapi-generator-cli.jar
 rm -Rf src/app/shared/openapi
 
-if [ "$npm_package_config_use_circle" = true ]
+if [ "$npm_package_config_use_artifactory" = true ]
 then
-        OPENAPI_PATH=$(./scripts/get-circleci-artifact-url.sh "$npm_package_config_circle_build_id" openapi.yaml)
+        # note that -DremoteRepositories=github-packages:https://maven.pkg.github.com/dockstore/dockstore also works with the right credentials, loaded as -s .github/snapshot-mvn-settings.xml
+        mvn dependency:get -DremoteRepositories=https://artifacts.oicr.on.ca/artifactory/collab-snapshot -Dartifact=io.dockstore:dockstore-webservice:${npm_package_config_webservice_version_prefix}-SNAPSHOT:openapi.yaml:dist -Dtransitive=false --batch-mode -ntp
```

Comment:
Presumably this fetches the latest snapshot? Looking at my local Maven repo, there are several SNAPSHOTS, each followed with a date/timestamp.

Reply:
This is my belief and intent, it seems a little poorly documented, but I think that is what is supposed to happen. 

For example, the second answer in https://stackoverflow.com/questions/5901378/what-exactly-is-a-maven-snapshot-and-why-do-we-need-it